sdk/src/types/block/output/feature/block_issuer.rs

```diff
@@ -48,6 +49,18 @@ impl BlockIssuerFeature {
     }
 }
 
+impl Rent for BlockIssuerFeature {
+    fn build_weighted_bytes(&self, builder: &mut RentBuilder) {
+        builder
+            // Feature Type
+            .data_field::<u8>()
+            // Expiry Slot
+            .data_field::<SlotIndex>()
+            // Public Keys
+            .packable_data_field(&self.public_keys);
```

Comment:
The keys here should be weighted with `block_issuer_key`

Reply:
God damn it